class-13.md

```diff
@@ -0,0 +1,138 @@
+
+### Class 13 - More Hyperledger Examples 
+
+We got started with private blockchains last week, exploring them through Hyperledger Composer and Fabric. Now we'll go a bit more in depth and look onto some specific code examples further.
+
+
+
+Reading & Videos:
+
+- [Composer Sample Networks](https://github.com/hyperledger/composer-sample-networks)* - This is a set of composer examples from the hyperledger organization. Try running one to see how it works, and get some ideas for how to implement solutions to problems in a permissioned blockchain.
+- [Fabric Insurance App](https://github.com/IBM/build-blockchain-insurance-app) - We'll look briefly at a full Hyperledger Fabric application, instead of just using the composer framework on top of it. This is a more in-depth example, showing how an insurance network, asset owners and the police could all interact in an automated claims process on the blockchain.
+
+### More Composer Examples
+
+Businesses have addressed a multitude of problems in a supply chain setup with software solutions in recent years, and blockchain can improve upon them. Let's take a look first at an identity example:
+
+**[Composer Identity Example](https://github.com/hyperledger/composer-sample-networks/tree/master/packages/pii-network)** - This shows how two users can add/revoke access to shared personally identifiable information.
+
+### Docker
+
+Before running our first fabric example, we'll run through some basic docker commands, go over what docker is and how to use it in a few basic configurations.
+
+Docker is just a lightweight VM that can be easily reproduced and shared, making it simple to reproduce environments between a developer's computer and a production server. It is also very useful for making each peer in a permissioned blockchain have a predictable environment for chaincode.
+
+#### Getting Started
+
+If you have installed composer locally, you should have docker and docker compose. If not, you can install it with these steps or with the following link:
+
+##### OSX
+
+```bash
+brew cask install docker
+```
+
+##### Ubuntu
+
+```bash
+sudo apt-get install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
+
+sudo apt-get update
+
+sudo apt-get install docker-ce
+```
+
+Similar to how Git has versioned states of code, Docker images are available in a similar repository. There is a service called DockerHub as well for sharing them, similar to GitHub. You can test out a basic example by running:
+
+```bash
+docker run hello-world
+```
+
+This will try to find an image called `hello-world`, and if it can't find it locally it will search through DockerHub for it. We'll find one, and play back a success message when it is successfully built.
```

Comment:
Revise to:
```
This command is used to find an image called `hello-world`. If the `hello-world` image does not exist locally (in the specified directory,) it will search through DockerHub for it. We'll find one, and play back a success message when it is successfully built.
```

> "We'll find one"
locally? Or through DockerHub? Please specify.